sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -807,6 +807,8 @@ class JDBCSuite extends QueryTest with SharedSparkSession {
       assert(doCompileFilter(LessThan(col0, 5)) === """"col0" < 5""")
       assert(doCompileFilter(LessThan(col0,
         Timestamp.valueOf("1995-11-21 00:00:00.0"))) === """"col0" < '1995-11-21 00:00:00.0'""")
+      assert(doCompileFilter(LessThan(col0,
+        LocalDateTime.of(2007, 12, 3, 10, 15, 30))) === """"col0" < '2007-12-03 10:15:30'""")
```

Comment:
I guess the suggestion also included to use `Timestamp.valueOf("2007-12-03 10:15:30"))`

Reply:
@amaliujia Are you talking about something like `Timestamp.valueOf("2007-12-03 10:15:30").toLocalDateTime` ?

Reply:
Yes. This is more readable. A readable string to generate to correct class instance is preferred in test code.